safe/gis/tools.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
+
+"""Tools for GIS operations."""
+
+from PyQt4.QtCore import QPyNullVariant
+
+from safe.definitions.layer_geometry import (
+    layer_geometry_raster,
+    layer_geometry_point,
+    layer_geometry_line,
+    layer_geometry_polygon,
+)
+from safe.utilities.gis import is_raster_layer, is_point_layer, is_line_layer
+
+
+__copyright__ = "Copyright 2016, The InaSAFE Project"
```

Comment:
2017

Reply:
I guess we missed a lot since January :)